content/projects/github-api-consume/_index.md

```diff
@@ -48,30 +48,45 @@ While for private repos a token will be required, the function should be able to
 
 Please be sure to follow the standard naming conventions for your language.
 
-Make sure the output matches the following example structure:
-```
+Make sure the output matches the following when called with the arguments below:
+
+```bash
 # input
 get_pull_requests("Umuzi-org", "ACN-syllabus", "2022-03-01", "2022-03-10")
 
 # output
 [
-  {"id":876359209, "user":"FaithMo", "title":"added data sci and eng info", "state":"open", "created_at":"2022-03-10"},
+  {"id":"876359209", "user":"FaithMo", "title":"added data sci and eng info", "state":"open", "created_at":"2022-03-10"},
   {"id":"874927260", "user":"ry-oc", "title":"update sololearn python and all contentlinks etc", "state":"closed", "created_at": "2022-03-09"},
-  {"id":872630389, "user":"Andy-Nkumane", "title":"added clarity on python error raising", "state":"open", "created_at":"2022-03-07"},
+  {"id":"872630389", "user":"Andy-Nkumane", "title":"added clarity on python error raising", "state":"open", "created_at":"2022-03-07"},
   {"id":"872484561", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
   {"id":"872482562", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"open", "created_at":"2022-03-06"},
   {"id":"872481470", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
   {"id":"872480774", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
-  {"id":"872480210", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"}
+  {"id":"872480210", "user":"Kate-bit-dev", "title":"Update _index.md", "state":"closed", "created_at":"2022-03-06"},
+  ...
 ]
 ```
 
+### Pagination
+
+**Pagination** is a process that is used to divide a large dataset into smaller chunks (pages).
+
+It is there to allow us to efficiently query through large data a chunk (page) at a time.
```

Comment:
"It is there to allow us to efficiently query through large data 1 page at a time." sounds DRYer